include/event2/dns.h

```diff
@@ -462,10 +467,10 @@ void evdns_cancel_request(struct evdns_base *base, struct evdns_request *req);
 
     ndots, timeout, max-timeouts, max-inflight, attempts, randomize-case,
     bind-to, initial-probe-timeout, getaddrinfo-allow-skew,
-    so-rcvbuf, so-sndbuf.
+    so-rcvbuf, so-sndbuf, use-vc, ignore-tc.
 
   In versions before Libevent 2.0.3-alpha, the option name needed to end with
-  a colon.
+  a colon. Options without values (use-vc, ingore-tc) shouldn't contain colon.
```

Comment:
Had to add this comment. I think it is not a good idea to allow colon for this options (options without values) since it might cause that such string `options use-vc:some_garbage` in `resolv.conf` will be a valid configuration.

Reply:
Agree